### ISSUE TITLE
Add the ■ char

### ISSUE THREAD

**Issue by asensio-project** [phip1611/noto-sans-mono-bitmap-rs#3]
Hello, 

I think this char ('■'), would be good for indicating that a char is not printable.
For adding it it's only necessary to put all the slice in 255.

Thanks.

**Comment by phip1611** [comment]
Hi! I'm currently on vacation and will come back to this at the end of
August.

asensio-project ***@***.***> schrieb am Mi., 3. Aug. 2022,
17:07:

**Comment by phip1611** [comment]
Do you mean, the character should be supported at all, or that it should be the default if a certain character is not available? @asensio-project

**Comment by phip1611** [comment]
I do not like the second one as it forces a policy on every user of the library. However, if the character is supported by the lib, you can do something like:


```rust
let bitmap_char = get_bitmap('😀', FontWeight::Regular, BitmapHeight::Size14).unwrap_or(
    get_bitmap('■', FontWeight::Regular, BitmapHeight::Size14).expect("fallback must exist")
);
```

**Comment by asensio-project** [comment]
Ok, that was my first idea.

**Comment by phip1611** [comment]
Will look into it on Monday. Currently, my library only includes whole unicode ranges. I need to check how I can marry this with additional stand-alone characters.

**Comment by asensio-project** [comment]
For a kernel based os.phil-opp.com.

**Comment by asensio-project** [comment]
How is it going? Can I help?

**Comment by phip1611** [comment]
I'm working on it. Thanks! I decided to do this in two steps:
1) a rather quick & dirty solution to my code generation code that  adds the functionality to add certain characters easily 
2) a refactoring that supports various unicode ranges via cargo build features (breaking change probably, combined with other changes)

**Comment by phip1611** [comment]
For the record: the symbol belongs to 

25A0 — 25FF  	[Geometric Shapes(https://jrgraphix.net/r/Unicode/25A0-25FF)

**Comment by phip1611** [comment]
published as v0.1.6
